articles/tech-resources/custom-bounce-domain.md

```diff
@@ -4,26 +4,50 @@ redirect_from: "https://support.sparkpost.com/customer/portal/articles/2371794-c
 ---
 
-You can improve your deliverability and email branding by adding a custom bounce domain. A custom bounce domain provides another layer to the branding that some ISPs care about.
+You can improve your deliverability and email branding by adding a bounce domain to each of your sending domains. A bounce domain provides another layer to the branding that some ISPs care about.
 
-**Note**: Only one custom bounce domain is permitted per account at this time.
+By using your sending domain as your bounce domain, you can customize the address that is used for the Return-Path header (which denotes the destination for out-of-band bounces). This bounce domain overrides the default Return-Path (also known as the envelope FROM) value of "sparkpostmail.com" for all messages sent.
 
-By adding a bounce domain to your account, you can customize the address that is used for the Return-Path header (which denotes the destination for out-of-band bounces). This custom bounce domain overrides the default Return-Path (also known as the envelope FROM) value of "sparkpostmail.com" for all messages sent.
+To configure a bounce domain, choose the subdomain you would like to use (e.g. mail.yourdomain.com) and add the following CNAME record to your DNS settings:
 
-To configure a custom bounce domain, choose the subdomain you would like to use (e.g. bounces.yourdomain.com) and add the following CNAME record to your DNS settings:
 
-| Name | Type | Data |
+| Hostname | Type | Value |
 | --- | --- | --- |
-| bounces.yourdomain.com | CNAME | sparkpostmail.com |
+| mail.yourdomain.com | CNAME | sparkpostmail.com |
 
-This DNS record will ensure that out-of-band/asynchronous bounces to be routed to SparkPost for processing.
+This DNS record will ensure that out-of-band/asynchronous bounces are routed to SparkPost for processing.
 
-Once you have configured your DNS settings, configure custom bounce domains through the SparkPost user interface, follow the steps below:
+Once you have configured your DNS settings, register and verify the domain with  SparkPost by following the steps below:
+
+1. Register the domain with SparkPost by using the sending domains [create endpoint](https://developers.sparkpost.com/api/sending-domains.html#sending-domains-create-post)
+1. CNAME-verify the domain by using the sending domains [verify endpoint](https://developers.sparkpost.com/api/sending-domains.html#sending-domains-verify-post).  Here's an example API call to CNAME-verify the mail.example.com domain:
+
+  ```
+  POST /api/v1/sending-domains/mail.example.com/verify
+
+  {
+    "cname_verify" : true
+  }
+  ```
+
+1. Optionally set the domain as your account default bounce domain so that it is automatically used as the bounce domain for all messages sent through the account.  Here's an example API call to set mail.example.com as the account default bounce domain:
```

Comment:
The numbering here results in a "1" rather than the intended "3" when viewing the rendered markdown in github.